_posts/2022-04-04-grant-announcements.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Announcing Brink's Latest Round of Developer Grants"
+permalink: /blog/2022/04/04/grantees/
+layout: post
+author: brink
+name: "brink"
+alt: Brink announces latest round of developer grants
+category: "News"
+description: Brink announces the latest round of developer grants.
+---
+
+Brink is proud to announce the recipients of its latest round of open source developer grants.
+After an open grant application process, the [Brink board of directors][brink
+board] has selected four developers who have demonstrated continued commitment to
+open source Bitcoin development. The grant recipients are:
+
+## "0xB10C"
+
+<img src="/assets/images/0xb10c.png" alt="Picture of 0xb10c" style="float:left;border:1px solid
+darkgray;margin-right:25px;width:150px;" />
+
+0xB10C writes of his Bitcoin experience and goals:
+
+I am a Bitcoin developer curious about what is happening on the Bitcoin
+network. Over the past years, I've built tools to visualize and observe, for
```

Comment:
Nit: the "for example" can be dropped from this sentence to simplify it.  (The example part is especially unneeded when the sentence ends with "and other protocol level statistics".)